adapters/repos/db/helpers/file_utils.go

```diff
@@ -9,11 +9,11 @@
 //  CONTACT: hello@weaviate.io
 //
 
-package lsmkv
+package helpers
 
 import "os"
 
-func fsync(path string) error {
+func Fsync(path string) error {
```

Comment:
IMO should instead be moved to [/entities/diskio/files.go](https://github.com/weaviate/weaviate/blob/main/entities/diskio/files.go)